ui/src/apiContext.ts

```diff
@@ -71,7 +71,8 @@ class FakeUnderlaysApi {
         },
         {
           type: "attribute",
-          title: "Year at birth",
+          id: "tanagra-year_of_birth",
```

Comment:
Can you use the attribute+entity as an id instead of manually specifying one here?

Reply:
No, there's no requirement for there to be only one of these for a given entity+attribute.